DocSearch.ipynb

```diff
@@ -0,0 +1,145 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook is a demonstration for how to use ThirdAI's document search engine. It assumes you are running in the ThirdAI demo docker container. Once in the container, you can get this notebook by running\n",
```

Comment:
Might be nice to list the sections of this notebook, e.g. 1. generating centroids, 2. building index, etc. just to give a high level overview of what's involved in building this system

Reply:
Sure, good idea